pycat/esd/methods.py

```diff
@@ -125,7 +125,7 @@ def relative_sdm(
         index = tuple(index_list)
 
         # consider only cells with valid observational data
-        if obs_cube_mask and obs_cube_mask[index]:
```

Comment:
`obs_cube_mask` is the return value of  `np.ma.getmask` which will be a `np.bool_` if the original array had no mask. Probably your suggestion is more robust.